.github/workflows/nuget_action.yml

```diff
@@ -0,0 +1,41 @@
+name: Nuget Release
+
+on:
+  create:
+    tag:
+      - '*.*.*'
+    
+  
+jobs:
+  package:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Get the version
+      id: get_version
+      run: echo ::set-output name=VERSION::$(echo $GITHUB_REF | cut -d / -f 3)
+    - name: Setup .NET
+      uses: actions/setup-dotnet@v1
+      with:
+        dotnet-version: 5.0.x
+    - name: Restore dependencies
+      working-directory: ./RiotSharp
+      run: dotnet restore
+    - name: Build    
+      working-directory: ./RiotSharp
+      run: dotnet build --no-restore      
+    - name: Build Nuget
+      working-directory: ./RiotSharp
+      run: dotnet pack -p:Configuration=Release -p:Version=${{ steps.get_version.outputs.VERSION }} --output ../npkgs
+    - name: Check npkg folder
+      working-directory: ./npkgs
+      run: ls
+    - name: Publish Nuget on nuget.org
+      working-directory: ./npkgs
+      run: dotnet nuget push RiotSharp.${{ steps.get_version.outputs.VERSION }}.nupkg -s https://nuget.org -k ${{secrets.NUGET_API_KEY}}
+    - name: Publish Nuget on github
+      working-directory: ./npkgs
+      run: dotnet nuget push RiotSharp.${{ steps.get_version.outputs.VERSION }}.nupkg -s https://nuget.pkg.github.com/BenFradet/index.json -k ${{secrets.MY_GITHUB_KEY}}
```

Comment:
I'm assuming this should be a github api token, do you know what scopes it needs?

Reply:
NUGET_API_KEY is described in markdown file.  
`MY_GITHUB_KEY` is outdated, GITHUB_TOKEN should do the job and this gets shipped with every action with the needed acces by default (uploading packages)

Reply:
Added an explaination in markdown regarding GITHUB_TOKEN